docs/subnets/create-a-subnet.md

```diff
@@ -172,3 +172,31 @@ Output:
 >> Registering subnet...
 ✅ Registered subnetwork with netuid: 1 # Your subnet netuid will show here, save this for later.
 ```
+
+:::caution A New Subnet is not Automatically Active
+To allow subnet owners to fully set up their subnets and to prevent extraction of emissions to subnet participants before the subnet is contributing to the network, new subnets are inactive and cannot be started for 7 * 7200 blocks (roughly one week) after they are registered.  During this time, you can register and activate validators and invite miners into the subnet.  **Be advised that the subnet and its participants will receive NO EMISSIONS during the time that the subnet is inactive.** 
+:::
+
+:::tip Validating in Your Own Subnet
+Ensure that you are meeting the [minimum requirements for validation](../validators#requirements-for-validation)  Another option for subnet owners is to ask one of the root network (subnet 0) validators to parent your validator hotkey as a childkey of theirs.  This will lend their stake to your validator, and can help you ensure that your validator maintains a sufficient stake to effectively participate in consensus as well as resist deregistration. See the [Child Hotkeys](/child-hotkeys) documentation for more detail.
```

Comment:
```suggestion
Ensure that you are meeting the [minimum requirements for validation](../validators#requirements-for-validation)  Another option for subnet owners is to ask one of the root network (subnet 0) validators to parent your validator hotkey as a childkey of theirs.  This will lend their stake to your validator, and can help you ensure that your validator maintains a sufficient stake to effectively participate in consensus as well as resist deregistration. See the [Child Hotkeys](./child-hotkeys) documentation for more detail.
```